stacyp2006/IdeaBox
# Search functionality

#### What's this PR do?

- Adds an queryselector for the search input field
- Adds a function linked to the search input field to our key-handler event listener
- Builds one more function which displays only cards that include the text within the search field (in either the title or body)

#### Where should the reviewer start?

main.js - line 115

#### How should this be manually tested?

build cards using the title and body input fields and the 'save' button. Then type some text into the search input field and check to make sure the cards update properly
